__tests__/index.test.ts

```diff
@@ -66,7 +66,7 @@ describe('Run', () => {
       },
       headless: false,
       screenshots: true,
-      filmstrips: false,
+      trace: false,
```

Comment:
Why did we remove the filmstrips option? It feels like these two things should be separate. I know there are similarities in the implementation but from a command line standpoint they do feel separate

Reply:
Brought it back.